index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingresar nombre')
+alert('Hola ' + nombreUsuario);
```

Comment:
Comentario de prueba
_En este caso se podria cambiar el saludo..._

```suggestion
alert('Bienvenido' + nombreUsuario);
```